### PR TITLE
Refactored log in page

### PR DESCRIPTION
**Pull request by joaojusto**
Using themeProvider to put custom color on the angular-material components, no need to create or override components to give the look we want.

### CODE REVIEW

File: app/assets/stylesheets/utilities/_margin.css.sass [hunk 1]
```diff
@@ -0,0 +1,7 @@
+@import '../components/theme'
+
+.u-margin
```
Comment (gabrielpoca) [inline app/assets/stylesheets/utilities/_margin.css.sass]: maybe defaultMargin, but just margin isn't right.

File: app/assets/stylesheets/utilities/_text_utilities.css.sass [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+.u-gamma
+  @include gamma
+
+.u-delta
+  @include delta
+
+.u-bold
+  @include theme-font-variation('bold')
+
+.u-whiteSpace
```
Comment (gabrielpoca) [inline app/assets/stylesheets/utilities/_text_utilities.css.sass]: This also doesn't work.

Comment (joaojusto) [inline app/assets/stylesheets/utilities/_text_utilities.css.sass]: What do you suggest then?

File: app/assets/stylesheets/utilities/_margin.css.sass (hunk 1 above)
Comment (joaojusto) [inline app/assets/stylesheets/utilities/_margin.css.sass]: Ok

File: app/assets/stylesheets/components/_theme.css.scss [hunk 3]
```diff
@@ -116,6 +116,10 @@ body {
     @include delta;
     line-height: 0.55;
   }
+
+  a {
```
Comment (joaojusto) [inline app/assets/stylesheets/components/_theme.css.scss]: I think there is no difference, it's more a question of what makes more sense, for me I think here is a good place to set the color of the links, I would left the reset for things like margins, borders and so on

File: app/assets/stylesheets/utilities/_margin.css.sass [hunk 4]
```diff
@@ -0,0 +1,7 @@
+@import '../components/theme'
```
Comment (joaojusto) [inline app/assets/stylesheets/utilities/_margin.css.sass]: You are right, the order is wrong, I'll fix that

File: app/assets/stylesheets/utilities/_base.css.sass [hunk 5]
```diff
@@ -1,3 +1,6 @@
 @import 'upper_case'
 @import 'fixed_top'
 @import 'full_size'
+@import 'text_utilities'
+@import 'center'
```
Comment (gabrielpoca) [inline app/assets/stylesheets/utilities/_base.css.sass]: maybe the problem is with `text_utilities`

File: app/assets/stylesheets/components/_theme.css.scss (hunk 3 above)
Comment (gabrielpoca) [inline app/assets/stylesheets/components/_theme.css.scss]: That's still not a good explanation to have two places. Is it just random? Are there no rules?

Comment (joaojusto) [inline app/assets/stylesheets/components/_theme.css.scss]: We didn't talk about it yet, maybe it's better to discuss in person

**Comment by gabrielpoca** [comment]
:+1:

**Pull request merged**